source/sigv4.c

```diff
@@ -3291,6 +3293,8 @@ SigV4Status_t SigV4_GenerateHTTPAuthorization( const SigV4Parameters_t * pParams
      * Note that the StringToSign starts from the beginning of the processing buffer. */
     if( returnStatus == SigV4Success )
     {
+        /* More details at: https://github.com/aws/SigV4-for-AWS-IoT-embedded-sdk/blob/main/MISRA.md#rule-182 */
```

Comment:
Add this line above as well:

```
        /* MISRA Ref 18.2.1 [Pointer subtraction within array] */
```

Reply:
See here for reference: https://github.com/aws/SigV4-for-AWS-IoT-embedded-sdk/blob/e82835329069d804e0db883d69eb4e1124dd7f08/source/include/sigv4.h#L67